blob/memstore/memstore.go

```diff
@@ -43,10 +91,10 @@ func compareEntries(a, b entry) int { return strings.Compare(a.key, b.key) }
 
 // Opener constructs a memstore, for use with the store package.  The address
 // is ignored, and an error will never be returned.
-func Opener(_ context.Context, _ string) (blob.KV, error) { return New(), nil }
+func Opener(_ context.Context, _ string) (blob.KV, error) { return NewKV(), nil }
```

Comment:
Eventually I want to change this interface to return stores instead of KVs, but for now if I do so it will make the downstreaming really painful.

The "store" package it speaks of actually no longer exists, it moved to be the library package in the ffstools repo 😓

Nonetheless, I made the comment less bad, and will fix it up more later.